app/views/application/home.html.erb

```diff
@@ -1,2 +0,0 @@
-<h1>Welcome!</h1>
-<%= link_to 'Your images', controller: 'images' %>
```

Comment:
Is there a controller action and tests that could be removed too?

Reply:
Yes, in a sense.  We did have a test that was meant to test this home page, but because we now use the index as the home page, it was (redundantly) testing that.  Thanks for prompting us to consider that.